examples/readme.md

```diff
@@ -0,0 +1,39 @@
+# Yew Examples
+
+Use `build.sh` script to build examples. 
+
+Examples are prepared for `wasm-bindgen` except `minimal_wp` which is prepared for `wasm-pack`.
+
+Note: `multi_thread` example has separate build script because it has app and worker ( two .wasm files )
+
+## How to run
+
+```
+git clone https://github.com/yewstack/yew.git
+cd yew
+examples/build.sh minimal # example subfolder
```

Comment:
Does the build script need to be run from the root of the repo? I think it would be nice if it could be run from anywhere.

Reply:
script can be run from anywhere, but I had a bug there - TARGET_DIR path was wrongly set